templates/taxonomy-post_format-post-format-link.html

```diff
@@ -0,0 +1,41 @@
+<!-- wp:template-part {"slug":"header","area":"header","tagName":"header"} /-->
```

Comment:
```suggestion
<!-- wp:template-part {"slug":"header"} /-->
```

Reply:
I don't think is going to be the final template. 🤷‍♀️